security/pkg/pki/util/generate_csr.go

```diff
@@ -61,6 +61,9 @@ func GenCSRTemplate(options CertOptions) (*x509.CertificateRequest, error) {
 		if err != nil {
 			return nil, err
 		}
+		if options.IsDualUse {
```

Comment:
We should not support multiple values in options.Host.
Could you remove this loop, and change the comments for options.Host, so that Host has a single value?

Reply:
Is this comment on the right line? not sure what loop you mean.  
I see there's a loop inside `BuildSubjectAltNameExtension` allowing multiple comma-delimited hosts, do you mean we should remove that functionality from Citadel in general? There seem to be a few tests testing exactly that. 

I can see a few other usages here:
- CA-server mode uses multiple hostnames for itself depending on the gRPC hostname flag
- generate_cert and generate_csr utils under `istio/security/tools/`
- `append-dns-names` flag in Citadel leads to multiple hostnames in `SecretController.generateKeyAndCert` (the whole `webhooks` block in `istio_ca/main.go`)